jenkins-builds/production.sh

```diff
@@ -159,6 +159,9 @@ fi
 # --- BUILD ---
 # load module EasyBuild-custom
 module load EasyBuild-custom/cscs
+if [[ "$system" =~ "dom" ]]; then
+ export EASYBUILD_EXTERNAL_MODULES_METADATA=${EB_CUSTOM_REPOSITORY}/cray_external_modules_metadata.cfg-19.10
```

Comment:
Since `cfg` is the extension of the file, I would write the filename as `cray_external_modules_metadata-19.10.cfg`. Perhaps my comment should have been to another PR, but I have just spotted this.